app/assets/stylesheets/layout_components/buttons/buttons.scss

```diff
@@ -91,7 +91,7 @@
 
     @include breakpoint(small down){
       display: block;
-      margin: 0 auto $base-margin auto;
+      margin: 0 auto;
```

Comment:
will this affect other apps?

Reply:
woops, that was actually a change I made and accidentally must have ctrl+z it.

Reply:
It just applies some margin bottom to our buttons on small screens, because when we have more than one button they are right up against each other.